MidasCivil_Engine/Convert/ToBHoM/Properties/ToConstraint6DOF.cs

```diff
@@ -94,6 +94,7 @@ public static Constraint6DOF ToConstraint6DOF(this string support, string versio
                                     }
                                 }
                             }
+                        support1 = delimitted[15].Trim();
```

Comment:
`supportName = delimitted[21].Trim(); `
For version = 8.8.5 the name field index is 21.

Make sure you check the formatting of the code, as it appears to be outside your `case `statement.